dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/transport/http/URLConnectionHTTPConduit.java

```diff
@@ -69,6 +70,9 @@
         DEFAULT_USE_REFLECTION = Boolean.valueOf(SystemPropertyAction.getProperty(HTTPURL_CONNECTION_METHOD_REFLECTION, "false"));
         SET_REASON_PHRASE = Boolean.valueOf(SystemPropertyAction.getProperty(SET_REASON_PHRASE_NOT_NULL, "false"));
     }
+    
+    private Boolean allowFixedLengthStreamMode = null;
+    private Message message = null;
```

Comment:
This seems unsafe.  What if you have multiple messages go out on the same conduit - iiuc, the more recent will overwrite the less recent.